package modelo.Exceptions;

public class MonstruoNoPuedeAtacarDosVecesEnUnTurnoException extends RuntimeException {
}
